docs/ref/models/database-functions.txt

```diff
@@ -286,10 +286,12 @@ Date Functions
 We'll be using the following model in examples of each function::
 
     class Experiment(models.Model):
-        start_time = models.DateTimeField()
+        start_datetime = models.DateTimeField()
```

Comment:
It would be nice not to rename all these fields, but if we must do it, let's make the changes in a separate commit so we can backport to 1.10.